modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -4,49 +4,49 @@
+=== Runtime Fabric 1.1.153 or later with Ingress
 
-This example shows a Runtime Fabric named "rtf231", which has an ingress that is using a certificate and private key in the secret group “alphatoken”. If you already have a Runtime Fabric and Secret Group configured, use those names in places of the ones in the example.
+This example shows a Runtime Fabric named "rtf231", which has an ingress that is using a certificate and private key in the secrets group “alphatoken”. If you already have a Runtime Fabric and secret group configured, use those names in places of the ones in the example.
 
 For more information, see xref:1.0@runtime-fabric::enable-inbound-traffic.adoc[Runtime Fabric ingress configuration].
 
 The image below shows an example.
 
 image::tokenization-example-rtf-ingress-config.png[]
 
-=== Secret Group
+=== Secrets Group
 
-Use Secrets Manager in the Anypoint platform Management Center to create and manage secret groups. For each tokenization service you create, you should also create a corresponding secret group in Secrets Manager to be used for that tokenization service only. No other tokenization service should use that secret group and no other secrets should be stored in that secret group.
+Use Secrets Manager in the Anypoint Platform Management Center to create and manage secrets groups. For each tokenization service you create, you must also create a corresponding, dedicated secrets group in Secrets Manager. No other tokenization service should use this secrets group and no secrets outside of this tokenization service should be stored in that secrets group.
 
 When you delete a tokenization service, you can delete the corresponding secret group.
 
 This is the recommended approach for several reasons:
 
-. Since this secret group should explicitly be used for tokenization service, it will avoid other secrets from being deleted accidentally on delete of secret group. 
-. You can create as many formats as you want and the secret group will never be exhausted. You can change the set of formats assigned to that tokenization service as often as you want and it will not exhaust the secret group.
-. This unique association between the tokenization service and its secret group will prevent running out of the secrets of type VDP Context in a secret group. In the worst case, there will be 10 secrets created of type VDP Context, one for each VDP domain.
+. Using a dedicated secrets group for the tokenization service helps avoid other secrets from being deleted accidentally when you delete a secrets group. 
+. You can create as many formats, and change the set of formats assigned to the tokenization service, and the secrets group will never be exhausted.
+. This unique association between the tokenization service and its dedicated secrets group prevents running out of the secrets of the VDP Context type in a secrets group. In the worst case, there will be 10 secrets created of the VDP Context type--one for each VDP domain.
```

Comment:
I don't really understand what all this means. Where do we spell out the limits on types in a secrets group? What about other types? Why is VDP Context special? Is this architectural info that belongs somewhere else? Or task-related info that should be in the topic where people create the association instead of this concept section?